io-ballerina/src/io/tests/unit-tests/io-test.bal

```diff
@@ -0,0 +1,335 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerina/java;
+
+@test:BeforeEach
+function beforePrint() {
+    resetOutputStream();
+    initOutputStream();
+}
+
+@test:Config {
+    dependsOn: ["testReadString"]
+}
+function testPrintAndPrintlnString() {
+    // output is equal to s1\ns2
+    string s1 = "Hello World...!!!";
+    string s2 = "A Greeting from Ballerina...!!!";
+    string expectedOutput = s1 + "\n" + s2;
+    println(s1);
+    print(s2);
+    test:assertEquals(readOutputStream(), expectedOutput, "Found unexpected output");
+}
+
+@test:Config{
+    dependsOn: ["testPrintAndPrintlnString"]
+}
+function testPrintAndPrintlnInt() {
```

Comment:
IMO, isn't it better to have a single test for all the parameter types (eg: `testPrint` has different assertions for string, int, float, boolean etc.) and the error message can be differentiated based on the parameter type.

Reply:
ATM we are reading console for each test. So, having multiple tests would be easy to find when an error occurs rather having it in a single test.

Reply:
Ack.